qds_sdk/cluster.py

```diff
@@ -24,7 +24,7 @@ class Cluster(Resource):
     """
 
     rest_entity_path = "clusters"
-    api_version = "v1.2"
```

Comment:
This api_version is useless I guess. 

We pass the appropriate version from `qds.py`.